bayesian/utils.py

```diff
@@ -358,3 +358,10 @@ def retrieve_worker_result (rdb, external_request_id, worker):
         result = row.to_dict()
     return result
 
+def get_request_count(rdb, external_request_id):
+    try:
+        count = rdb.session.query(StackAnalysisRequest).filter(\
+                    StackAnalysisRequest.id == external_request_id).count()
+        return count
+    except SQLAlchemyError:
```

Comment:
I think this try-except block is not needed here as we would like to get these kind of errors in logs.